params/dhcpd-server.conf

```diff
@@ -3,7 +3,5 @@
 # Add subnet declaration for the server
 # Define range of addresses that the server can assign to other devices
 subnet 192.168.2.0 netmask 255.255.255.0 {
-	max-lease-time 60;
```

Comment:
Do you know what the default is? I found something saying it is 86400, but we should be sure

Reply:
it's like 5 or 10 minutes or some hours, I don't know but I believe it doesn't matter. This was here, so that the surface laptop could be hotplugged, because we were serving only one dhcp address, we needed to make it re-available

Reply:
I think it just can't be forever. If there is a value, it will be fine.

Reply:
the default is 24 hours.

Reply:
I'll change it to 10 minutes

Reply:
hmmm. what happens if the lease expires with the computer connected? I never looked closely at that. 24h sounds good. I don't expect anyone to connect 10 different devices to it in under 24h.

Reply:
> what happens if the lease expires with the computer connected?

The dhcp server and client do their negotiations, and the user notices nothing.

Reply:
10min is fine too.

Reply:
is this re-negotiation properly implemented in ping360 side? can you check?

Reply:
I will check. It appeared to be ok between reboots and so on. If it's not ok on ping360 side, there's not much we can do about it right here.

Reply:
@Williangalvani it's fine, keeps going. I tried 30seconds